### PR TITLE
Feat/one alias per environment

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |N|
|New feature?   |Y|	
|BC breaks?     |Y|	
|Deprecations?  |Y|	
|Fixed tickets  |N|	

After reflexion rollback to the feature allowing the define multiple instance_id in a skeleton config. And handling complex code with an array of aliases.

Now the environment  names from the EMSCH_ENV and EMSCH_ENVS variables) should match an alias. Up to the authentic sources to register their indexes into that alias. 

Obviously for website managed by a single elasticms/source you can use the same alias. 

Note, if you have multiple skeleton environments for the same elasticms environment you'll have to define 1 alias for each locale, on to of the elasticms one.

I.e. you have an elasticms indexing content in an index named my_content_preview_20201220_121212. Elasticms automaically associated that index to an aliases my_content_preview. If your web site have one domaine per locales you'll have to associate the index my_content_preview_20201220_121212 also with the aliases my_website_preview_en and my_website_preview_fr> And than the EMSCH_ENVS looks like:
```json
{
  "my_website_preview_en": {
    "cache_environments": [
      "preview"
    ],
    "cache_instance_ids": [
      "my_content_"
    ],
    "regex": "/.*/",
    "request": {
      "_locale": "en"
    }
  },
  "my_website_preview_fr": {
    "cache_environments": [
      "preview"
    ],
    "cache_instance_ids": [
      "my_content_"
    ],
    "regex": "/.*mon-site-web.be.*/",
    "request": {
      "_locale": "fr"
    }
  }
}
```

Note the config options cache_environments and cache_instance_ids. They are mandatory and should contains the instance_id and the environment of the elasticmss managing your routes, labels and templates.

The fun with this approach, is we can now integrate some live content in a staging website if some authentic source doesn't have/need staging content.

### CODE REVIEW

**Comment by Davidmattei** [comment]
**Update**

The clientHelper is not using the logger index anymore for cache invalidation.
New environment config

```json
{
  "my_website_preview_en": { 
    "regex": "/.*/",
    "request": {
      "_locale": "en"
    }
  },
  "my_website_preview_fr": {
    "regex": "/.*mon-site-web.be.*/",
    "request": {
      "_locale": "fr"
    }
  }
}
```

File: Helper/ContentType/ContentTypeCollection.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+final class ContentTypeCollection
+{
+    /** @var array<ContentType> */
+    private array $contentTypes = [];
+
+    public static function fromResponse(string $alias, Response $response): ContentTypeCollection
+    {
+        $collection = new self();
+
+        if (null === $aggContentType = $response->getAggregation('contentTypes')) {
```
Comment (theus77) [inline Helper/ContentType/ContentTypeCollection.php:18]: define the string as CONST?

**Review by theus77 — COMMENTED** [review]
I can approve it, it's my PR. But It's ok

File: Helper/ContentType/ContentTypeCollection.php (hunk 1 above)
Comment (Davidmattei) [reply Helper/ContentType/ContentTypeCollection.php:18]: yes will do

Comment (Davidmattei) [reply Helper/ContentType/ContentTypeCollection.php]: Fixed 5178ab0

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**